doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -2759,6 +2795,14 @@ \subsubsection{Surface Outside Face Thermal Radiation to Ground Heat Transfer Co
 
 This is the coefficient that describes thermal radiation heat transfer between the outside face and the ground surrounding the surface. It is the value of ``Hr'' in the classic linearized model for thermal radiation Q = Hr * A * (T\_surf -- T\_odb) when applied to the ground. Where T\_surf = Surface Outside Face Temperature, and T\_odb = Site Outdoor Air Drybulb Temperature (used as an approximation for the ground surface temperature). Former Name: Prior to version 7.1, this output was called ``Surface Ext Rad to Ground Coeff.''
 
+\subsubsection{Surface Outside Face Thermal Radiation to Air Heat Transfer Rate {[}W{]}}\label{surface-outside-face-thermal-radiation-to-air-heat-transfer-rate-w}
+
+This is thermal radiation heat transfer rate between the outside face and the air mass surrounding the surface, calculated by the coeffient.
+
+\subsubsection{Surface Outside Face Heat Emission to Air Rate {[}W{]}}\label{surface-outside-face-heat-emission-to-air-heat-transfer-rate-w}
+
+This is total heat transfer rate between the outside face and the air mass surrounding the surface by convection and thermal radiation.
```

Comment:
Is this variable equal to "Surface Outside Face Thermal Radiation to Air Heat" minus "Surface Outside Face Convection Heat Gain Rate"? If so, state that here.

Reply:
Yes, it is.